frontend/src/components/pages/home-page.tsx

```diff
@@ -441,9 +443,8 @@ const MarimoFileComponent = ({
       </div>
       <div className="flex flex-col gap-1 items-end">
         <div className="flex gap-3 items-center">
-          <div>
-            <SessionShutdownButton filePath={file.path} />
-          </div>
+          <CopyNotebookButton filePath={file.path} />
```

Comment:
I think we can put this in notebook actions - so it automatically appears in the notebook dropdown and in the command palette. It doesn't need to be its own button.

Reply:
Sounds good! I was just going off the issue author's suggestions, thought it'd be a simple add

Reply:
Oh yea I didn't see the second part of the issue mentioning that. I don't know how common it'll be and does complicate the routes. I think we hold off homepage and consider later